versioned_docs/version-latest/protocol/security.md

```diff
@@ -1,86 +1,266 @@
 ---
-title: Safety and Security
-sidebar_position: 6
-description: Security and Scam Prevention
+title: General and server security guidelines
+sidebar_position: 5
```

Comment:
We should keep the sidebar position at 6 since now we've added the information for timekeepers that should come earlier.

Reply:
ok i see. So for my two pages -Security and Partitioning - I can count on slots 6 and 7 ?

Reply:
Let's do 6 and 8 please, 7 could be "Common problems"